pyblish_lite/model.py

```diff
@@ -397,7 +397,8 @@ def update_with_result(self, result):
         self.setData(index, False, IsIdle)
         self.setData(index, False, IsProcessing)
         self.setData(index, True, HasProcessed)
-        self.setData(index, result["success"], HasSucceeded)
+        if not self.data(index, HasSucceeded):
```

Comment:
Right, I forgot about this.
Some time ago I had an issue: I ran the plugins, an instance failed some of them and passed the rest, so it should be colored red, but was not, it was green. I've figured out that this is probably due to overriding this failed result with some later success, so that's how those lines emerged.

Right now I cannot recreate it and I'm not sure if I was right, so I'll just delete those lines.